### DISCUSSION TITLE
Passing periods while class active

### DISCUSSION THREAD

**Discussion by asportnoy** [chromezoneeths/ethsbell-rewrite#75] [Ideas]
During the passing periods between **Lunch 1 and Block 5B/6B** or **Block 5A/6A and Lunch 2**, there is another class going on. Because of this, the psusedo passing period is not created.

Should we manually define a passing period in the schedule files for this situation? Would this potentially create issues with some frontends/applications?

**Comment by allegedengineer** [comment]
I don't think anything would break because of that. It's also already possible to manually declare pseudo-periods in the schedules, so it won't require any backend changes. If you do that, you should also update the doc comment at src/schedule/period.rs:65 to include that use-case.
